spec/rails_helper.rb

```diff
@@ -53,6 +52,25 @@
   # For time travel.
   config.include ActiveSupport::Testing::TimeHelpers
   config.include GdsHelper
+
+  # Use a separate in-memory event store for each spec, instead of the
+  # event store configured by the initializer. Having a separate store
+  # for each spec reduces the surface area when working with events.
+  # And running an in-memory repository reduces hits to the database.
+  config.before do |_example|
+    Rails.configuration.event_store = RailsEventStore::Client.new(
+      repository: RubyEventStore::InMemoryRepository.new
+    )
+  end
+
+  # Subscribe all apart from notifier handler for system specs
+  config.before(:each, type: :system) do |_example|
+    event_store = Rails.configuration.event_store
+
+    CurrentAssignments::Configuration.new.call(event_store)
+    # ReceivedOnReports::Configuration.new.call(event_store)
```

Comment:
Is there a reason you have left this commented out?

Reply:
sorry. fixed